render/filters.go

```diff
@@ -58,12 +58,26 @@ func ColorConnected(r Renderer) Renderer {
 // Filter removes nodes from a view based on a predicate.
 type Filter struct {
 	Renderer
-	FilterFunc func(report.Node) bool
+	FilterFunc     func(report.Node) bool
+	ReportFiltered bool // false means we don't report stats for how many are filtered
```

Comment:
I see filtering as orthogonal to whether a node is a pseudo node or not.  As Peter might say, I think not reporting the number of filtered pseudo nodes as a category error.